.circleci/config.yml

```diff
@@ -44,22 +43,47 @@ jobs:
               sudo apt-get install libgl1-mesa-glx libegl1-mesa libxrandr2 libxrandr2 libxss1 libxcursor1 libxcomposite1 libasound2 libxi6 libxtst6 qt5-default
 
         - run:
-            name: Get Anaconda running
+            name: Get conda running
             command: |
               wget -q https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh;
               chmod +x ~/miniconda.sh;
               ~/miniconda.sh -b -p ~/miniconda;
-              export export PATH=~/miniconda/bin:$PATH;
-              conda update --yes --quiet conda;
-              conda env create -f environment.yml
+              echo "export PATH=~/miniconda/bin:$PATH" >> $BASH_ENV;
+
+        - run:
+            name: Make MNE conda environment and install BIDS packages
+            command: |
+              conda update -n base -c defaults conda
+              conda env create -f environment.yml --name mne
+              pip install --upgrade mne_bids
+              pip install --upgrade pybids
```

Comment:
I didn't realize that the circleci config had become so complicated. For `mne-bids` to work, you can just do a plain `$ pip install mne`. We don't need the `mayavi` features at the moment. No point in raising the barrier to entry ...